### PR TITLE
Docs:align header and main html elements

### PR DESCRIPTION
**Pull request by edjrodriguez**
## CSS
**styles.css**
- lines 1 - 61
- align header to center
- align swatch container to main container
- align hex code input and lock button to swatch
- aligned and add buttons - random and save

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="https://use.typekit.net/wng7kgg.css">
```
Comment (DinneK) [inline index.html:5]: Add file style approved

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
+* {
+  border: solid red 1px;
+  font-family: briller,sans-serif;
+  font-weight: 200;
+  font-style: normal;
+  /* height: 100%; */
+}
+/* body{
```
Comment (jrmedina) [inline styles.css:8]: We can remove this on the next merge

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  margin-bottom: 0;
+  margin-top: 0;
+} */
+body {
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  /* justify-content: space-between; */
```
Comment (jrmedina) [inline styles.css:25]: remove

File: index.html [hunk 4]
```diff
@@ -11,36 +12,36 @@ <h1>ColoRando</h1>
     </header>
     <main>
       <section class="swatch-container">
-        <article class="">
-          <input type="text" name="hex-code-1" value="hex-code">
+        <article class="swatch">
+          <input class="hex-code"class="hex-code" type="text" name="hex-code-1" value="hex-code">
```
Comment (DinneK) [inline index.html:16]: add class swatch

File: index.html (hunk 1 above)
Comment (GOECHA) [reply index.html:5]: - font type: briller - approved for header -CG

File: index.html (hunk 4 above)
Comment (GOECHA) [reply index.html:16]: - reviewed- class swatch - approved

File: index.html [hunk 5]
```diff
@@ -11,36 +12,36 @@ <h1>ColoRando</h1>
...
         </article>
       </section>
       <section class="user-input">
-        <div class="save-button">
-          <button type="button" name="button"></button>
+        <div class="save-button ">
+          <button type="button" name="button">Save Palette</button>
         </div>
         <div class="random-button">
-          <button type="button" name="button"></button>
```
Comment (GOECHA) [inline index.html:40]: - Need button style

File: styles.css (hunk 2 above)
Comment (GOECHA) [reply styles.css:8]: reviewed remove- approve CG

**Review by GOECHA — APPROVED** [review]
Reviewed changes- approved CG

**Review by DinneK — APPROVED** [review]

**Pull request merged**